inc/Engine/Preload/Controller/CheckExcludedTrait.php

```diff
@@ -23,28 +26,54 @@ public function add_cache_reject_uri_to_excluded( array $regexes ): array {
 	 * @return bool
 	 */
 	protected function is_excluded_by_filter( string $url ): bool {
+		/**
+		 * Regex to exclude URI from preload without sanitize.
+		 *
+		 * @param string[] regexes to check
+		 */
+		$regexes = (array) apply_filters( 'rocket_preload_exclude_urls_regexes', [] );
 		/**
 		 * Regex to exclude URI from preload.
 		 *
 		 * @param string[] regexes to check
 		 */
-		$regexes = (array) apply_filters( 'rocket_preload_exclude_urls', [] );
+		$regexes_urls = (array) apply_filters( 'rocket_preload_exclude_urls', [] );
 
+		if ( $this->check_regexes( $url, $regexes, false ) ) {
+			return true;
+		}
+
+		return $this->check_regexes( $url, $regexes_urls );
+	}
+
+	/**
+	 * Check if one of the regexes is matching the URL.
+	 *
+	 * @param string $url URL to test.
+	 * @param array  $regexes Regexes to try.
+	 * @param bool   $sanitize sanitize regexes.
+	 *
+	 * @return bool
+	 */
+	protected function check_regexes( string $url, array $regexes, bool $sanitize = true ): bool {
```

Comment:
```suggestion
	protected function is_match( string $url, array $regexes, bool $sanitize = true ): bool {
```

I think it's a better method name to know what this does